knative-operator/pkg/controller/knativekafka/knativekafka_controller_test.go

```diff
@@ -584,7 +584,8 @@ func TestDisabledControllers(t *testing.T) {
 	}}
 
 	for _, test := range tests {
-		defaultDeployment := makeEventingKafkaDeployment(t, "")
+		// by default we assume all disabled:
+		defaultDeployment := makeEventingKafkaDeployment(t, "broker-controller,trigger-controller,sink-controller")
```

Comment:
Do you think this fixes the problem?

Isn't the issue about [`StringValues`](https://github.com/openshift-knative/serverless-operator/blob/a482801e95f7b24a70f3e67ad92fc882fd1ca59e/knative-operator/pkg/controller/knativekafka/knativekafka_controller.go#L531) that might return results in a different order since [iterating a map](https://github.com/openshift-knative/serverless-operator/blob/a482801e95f7b24a70f3e67ad92fc882fd1ca59e/knative-operator/pkg/common/util.go#L30-L38) isn't guaranteed to be done in order?

Reply:
I know, but the order is irrelvant, on run time. This is really to fix the flake. as a work-around